src/inputs/TextFieldBase.tsx

```diff
@@ -58,6 +58,7 @@ export interface TextFieldBaseProps<X>
   hideErrorMessage?: boolean;
   // If set, the helper text will always be shown (usually we hide the helper text if read only)
   alwaysShowHelperText?: boolean;
+  unfocusedPlaceholder?: ReactNode;
```

Comment:
Suggestion: I can't immediately guess what this prop does; it does seem like a good name, but maybe add a comment describe what it does with an "i.e. for multi-select this (...the chips...?)"

Reply:
Good call, updated